verilog/CST/package.h

```diff
@@ -35,7 +36,7 @@ std::vector<verible::TreeSearchMatch> FindAllPackageImportItems(
     const verible::Symbol& root);
 
 // Extract the subnode of a package declaration that is the package name.
-const verible::TokenInfo& GetPackageNameToken(const verible::Symbol&);
+const verible::TokenInfo&  GetPackageNameToken(const verible::Symbol&);
```

Comment:
Only a spacing change in this file, so revert this one.

Reply:
done!